README.md

```diff
@@ -2,6 +2,7 @@
   <img src="/securedrop/static/i/logo.png" width="350" height="314">
 </p>
 
+> [There are many ways to contribute to SecureDrop, and we welcome your help!](CONTRIBUTING.md) By contributing to this project, you agree to abide by our [Code of Conduct](https://github.com/freedomofpress/.github/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
I would propose that we use the "By contributing to this project" part of this language in the README for all `freedomofpress` repos (SecureDrop or otherwise)